tools/bin/ariaDownload.py

```diff
@@ -307,6 +335,86 @@ def check_cookie_is_logged_in(cj):
             return True
     return False
 
+def rewrite_summary(infos):
+    succeed, failed, skipped, tot_time, tot_bytes = [], [], [], 0, 0
+    for info in infos:
+        if info[0]: succeed.extend(info[0])
+        if info[1]: failed.extend(info[1])
+        if info[2]: skipped.extend(info[2])
+        tot_time += float(info[3])
+        tot_bytes += float(info[4])
+
+
+    log.info ('Successes: %d files, %s bytes', len(succeed), tot_bytes)
+    _ = [print ('\n\t-', sf['file'], f'{sf["size"]/1024**2:.2f}') for sf in succeed]
+
+    if skipped: log.info('Skipped: %d files', len(skipped))
+    _ = [print ('\n\t-', sf) for sf in skipped]
+
+    if failed: log.info('Failures: %d files', len(failed))
+    _ = [print ('\n\t-', sf) for sf in failed]
+
+    if succeed: log.info('Average Rate: %.2f MB/sec', (tot_bytes/1024.0**2)/tot_time)
+
+    if failed:
+        log.critical('We recommend rerunning the same ariaDownload command to address the %d failures', len(failed))
+    else:
+        log.info ('All files have been downloaded successfully')
+    return
+
+def status_plot(wd, rates=None, use_all=False):
+    """
```

Comment:
Codacy found an issue: [1 blank line required between summary line and description (found 0) (D205)](https://app.codacy.com/gh/aria-tools/ARIA-tools/pullRequest?prid=6310391)